libdash/libdash/source/manager/DASHManager.cpp

```diff
@@ -34,6 +38,203 @@ IMPD*           DASHManager::Open   (char *path)
 
     MPD* mpd = parser.GetRootNode()->ToMPD();
 
+    std::cout << "Variables check:" << std::endl;
```

Comment:
That looks like some testing code that should not be part of the PR, right?
If so, please remove that whole part.

Reply:
You are right. It was part of the testing code. I have already provided to the code removal.